src/modes.js

```diff
@@ -25,17 +26,22 @@ module.exports = {
 
         // assign a power level list to each date
         var deviceLevels = [];
+        var deviceIDs = [];
 
         deviceMap.forEach(function(device){
             // console.log('dif', now.format() - Date.parse(device["Last time seen"]));
 
-            if (now - Date.parse(device["Last time seen"]) < interval * 1000)
+            if (now - Date.parse(device["Last time seen"]) < interval * 1000) {
                 deviceLevels.push(device.Power);
+                deviceIDs.push((new Buffer(crypto.createHash('sha256').update(device["Station MAC"]).digest())).readUInt32LE(0));
```

Comment:
Here, signal strength and device ids are split. They should be kept together.
They can be later split in protobuf for the sake of better encoding (and backward-compat reasons), but not at this point